security/cmd/istio_ca/main.go

```diff
@@ -160,11 +158,10 @@ func init() {
 	flags.StringVar(&opts.listenedNamespaces, "listened-namespaces", metav1.NamespaceAll,
 		"Select the namespaces for the Citadel to listen to, separated by comma. If unspecified, Citadel tries to use the ${"+
 			cmd.ListenedNamespaceKey+"} environment variable. If neither is set, Citadel listens to all namespaces.")
+	flags.BoolVar(&opts.manageUnlabeled, "manage-unlabeled", true, "Determines whether Citadel"+
+		"should generate secrets for service accounts created in namespaces without the ca.istio.io/managed label")
 	flags.StringVar(&opts.istioCaStorageNamespace, "citadel-storage-namespace", "istio-system", "Namespace where "+
 		"the Citadel pod is running. Will not be used if explicit file or other storage mechanism is specified.")
-	flags.BoolVar(&opts.explicitOptInRequired, "explicit-opt-in", false, "Specifies whether Citadel requires "+
```

Comment:
Unless this never shipped - it's not ok to remove flags without deprecating them for a release.

Reply:
it's never shipped or never supported officially... when @Monkeyanator tries to set it up, istio installation profile does not include necessary k8s rbac to make things work at all.